src/declarations/rewards/rewards.did

```diff
@@ -29,14 +42,18 @@ type AccountSnapshot_Spl = record {
 	amount : nat64
 };
 
-type AirDropConfig = record {
-	/// Max number of participants
-	number_of_participants : nat64;
-	/// When to start the timestamp
-	start_timestamp_ns : nat64;
-	/// The tokens to distribute during the airdrop
-	token_configs : vec TokenConfig
+/// A network on which a token can be found.
+type AnyNetwork = record {
+	network_id : text;
+	testnet_for : opt text
+};
+
+/// A token on a specified network.
+type AnyToken = record {
+	token_symbol : text;
+	wraps : opt text
```

Comment:
Note: It's vine to put in an arbitrary value such as "TODO" if it is hard to determine the upstream token.

The idea here is to capture groups such as {BTC, ckBTC} or various USD stablecoins.

Reply:
It would be nice to have namespacing, such as "FIAT::USD" or "ICP::ONE" or "ETH:56:BNB"

I might sound like a broken record.

But namespacing is needed in a multi-chain world.